app/controllers/sessions_controller.rb

```diff
@@ -24,7 +24,7 @@ def create
 
       render "multifactor_auths/prompt"
     else
-      do_login
+      do_login(two_factor_label: nil, two_factor_method: nil, authentication_method: "password")
```

Comment:
Maybe we could use a constant for the magic strings in here to keep it consistent. Minor nitpick.